src/lib/zkapps_examples/zkapps_examples.ml

```diff
@@ -569,19 +586,23 @@ let compile :
       | [] ->
           []
       | prover :: provers ->
+          (* here's the wrapping of prover *)
           let prover ?handler () =
             let open Async_kernel in
+            (* run the prover *)
             let%map ( _stmt
                     , ( { account_update; account_update_digest; calls }
                       , auxiliary_value )
                     , proof ) =
               prover ?handler ()
             in
+            (* create account update *)
             let account_update : Account_update.t =
               { body = account_update
               ; authorization = Proof (Pickles.Side_loaded.Proof.of_proof proof)
               }
             in
+            (* new result is Call_forest.Tree * auxiliary_value *)
```

Comment:
Use a type annotation if you think this is important to expose to future readers.